### PR TITLE
Feat default rules

### PR DESCRIPTION
**Pull request by franfaccin**
Create the 2 default rules of the project.

**Depends on PR https://github.com/vsfba1-iscteiul/ES1-2019-EIPL-81/pull/10**

### CODE REVIEW

**Comment by franfaccin** [comment]
[(1) [Story] Criar regras default](https://trello.com/c/Mqrqt4ti/9-1-story-criar-regras-default)

File: src/main/java/controller/MainController.java [hunk 1]
```diff
@@ -39,6 +40,17 @@
 	private ExcelImporter ei;
 	private ArrayList<String[]> excelRows;
 	private ArrayList<ExcelRow> excelRowsConverted = new ArrayList<ExcelRow>();
+	private ArrayList<CodeQualityRule> rulesList = new ArrayList<CodeQualityRule>();
+
+	/**
+	 * MainController constructor. Creates the default rules to be used.
+	 */
+	public MainController() {
+		CodeQualityRule is_long_method = new CodeQualityRule("is_long_method", "LOC > 80 &6 CYCLO > 10", true, false);
```
Comment (lrnsa-iscteiul) [inline src/main/java/controller/MainController.java]: A little mistake in the creation of the CodeQualityRule objects, you used &6 instead of &&

**Review by lrnsa-iscteiul — CHANGES_REQUESTED** [review]
See comment

**Review by lrnsa-iscteiul — APPROVED** [review]
It's perfect

**Review by djsar-iscteiul — APPROVED** [review]

**Pull request merged**